cypress/integration/tests/developer/reports/configure.report.test.ts

```diff
@@ -0,0 +1,19 @@
+import { login, preservecookies } from "../../../../utils/utils";
+import { ReportConfig } from "../../../models/developer/reports/reportConfig";
```

Comment:
Please rename it to download_report.test.ts

Reply:
Done